pootle/core/search/broker.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from . import SearchBackend
+
+import importlib
+
+from django.conf import settings
+
+
+class SearchBroker(SearchBackend):
+    def __init__(self, config_name=None):
+        super(SearchBroker, self).__init__(config_name)
```

Comment:
Better to pass `config_name` as kwargs here and elsewhere; you never know how the parent class could change.